setup.py

```diff
@@ -74,10 +70,38 @@
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Framework :: Jupyter",
+        "Framework :: Jupyter :: JupyterLab",
+        "Framework :: Jupyter :: JupyterLab :: 3",
+        "Framework :: Jupyter :: JupyterLab :: Extensions",
+        "Framework :: Jupyter :: JupyterLab :: Extensions :: Prebuilt",
         "Topic :: Scientific/Engineering :: Visualization",
     ],
 )
 
+# Representative files that should exist after a successful build
+ensured_targets = [str(lab_path / "package.json"), str(lab_path / "static/style.js")]
+
+try:
```

Comment:
I would make it simpler by adding something like:
```
try:
    from jupyter_packaging import wrap_installers, npm_builder, get_data_files
except ImportError as e:
   raise ImportError("Build tool `jupyter-packaging` is missing. Install it with pip or conda.")
```
with the top imports and then expect `jupyter_packaging` to be available (it's in `pyproject.toml`).
Not having  `jupyter_packaging` should only occurs with `python setup.py ...` when building from the source... so :man_shrugging: